.github/workflows/release.yml

```
@@ -38,6 +38,10 @@ jobs:
           - "3.8"
           - "3.9"
           - "3.10"
+        variant:
+          - buster
+          - slim
+          - alpine
```

I suggest indicating the default Python versions and variants to make our build steps more generic[^1].

```suggestion
          - alpine
        include:
          - python_version: "3.10"
            is_default_python_version: true
        include:
          - variant: "buster"
            is_default_variant: true
```

[^1]:  https://docs.github.com/en/actions/using-jobs/using-a-build-matrix-for-your-jobs#example-including-additional-values-in-combinations

---

👍 good catch!